__tests__/ic3/createAdapterEnhancer.spec.js

```diff
@@ -1,7 +1,8 @@
-import createIC3Enhancer from './../../src/ic3/createAdapterEnhancer';
-import { StateKey } from './../../src/types/ic3/IC3AdapterState';
-import * as initializeIC3SDK from './../../src/ic3/initializeIC3SDK';
 import * as getPlatformBotId from './../../src/ic3/utils/getPlatformBotId';
+import * as initializeIC3SDK from './../../src/ic3/initializeIC3SDK';
+
```

Comment:
remove this line?

Reply:
It is automatically added by VScode editor